lib/3.7/packages.cf

```diff
@@ -1229,8 +1229,8 @@ body package_method smartos
 {
       package_changes => "bulk";
       package_list_command => "/opt/local/bin/pkgin list";
-      package_list_name_regex    => "([^\s]+)\-[0-9]+.*\s";
-      package_list_version_regex => "[^\s]+\-([0-9][^\s]+)\s";
+      package_list_name_regex    => "([^\s]+)\-[0-9a-b.]+.*[; ]";
+      package_list_version_regex => "[^\s]+\-([0-9a-b.]+).*[; ]";
```

Comment:
I've updated the regex with your suggestion. I also ran it through an analysis to check against the existing package list for 2014Q4 which can be viewed [here][1] and checked for potential misses.

[1]: https://gist.githubusercontent.com/bahamat/7587dbee49a756f4773b/raw/pkgin_regex_matches